tests/unit/pricing/test_greeks_extended.py

```
@@ -0,0 +1,534 @@
+            STD_K,
+            STD_T,
+            STD_R,
+            STD_Q,
+            1.5,
+            OptionType.CALL,
+            ExerciseStyle.EUROPEAN,
+        )
+        assert math.isfinite(result)
+
+    def test_low_sigma_finite(self) -> None:
+        """Vomma remains finite with low volatility (above dσ threshold)."""
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF002)